### PR TITLE
Update set-up-mobile-policies-for-your-organization.md

### PR DESCRIPTION
**Pull request by pupara**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-policies-in-your-organization/set-up-mobile-policies-for-your-organization.md [hunk 1]
```diff
@@ -32,40 +32,23 @@ Mobile policy settings can be configured at the time a policy is created, or you
 > [!NOTE]
 > For all of the mobile policy settings in Skype for Business Online, you must use Windows PowerShell and you **can't use** the **Skype for Business admin center**. 
   
-### Verify and start Windows PowerShell
+### Start Windows PowerShell
 
-- **Check that you are running Windows PowerShell version 3.0 or higher**
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/set-up-mobile-policies-for-your-organization.md:37]:
```suggestion
> [!NOTE]
```

File: Skype/SfbOnline/set-up-policies-in-your-organization/set-up-mobile-policies-for-your-organization.md [hunk 2]
```diff
@@ -32,40 +32,23 @@ Mobile policy settings can be configured at the time a policy is created, or you
...
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/set-up-mobile-policies-for-your-organization.md:38]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**